pkg/utils/test/setup_helpers.go

```diff
@@ -214,7 +214,7 @@ func ExtractModuleVersion(moduleName string) (string, error) {
 // DeployCRDs deploys the CRDs commonly used in tests.
 func DeployCRDs(ctx context.Context, crdPath string, operatorClient *operatorclient.Clientset, cluster clusters.Cluster) error {
 	// CRDs for stable features.
-	kubectlFlags := []string{"--server-side", "-v5"}
+	kubectlFlags := []string{"--server-side", "-v5", "--force-conflicts"}
```

Comment:
I guess this flag has been added to perform local testing, but it shouldn't be required in the CI environment, right? If that's the case, can we put it behind an env var that defaults to false?

Reply:
👍 8428a3b33b001fb0f941b93824de7dcfc8ef9500